### PR TITLE
Rename geo to country

### PR DESCRIPTION
**Pull request by trink**

### CODE REVIEW

File: heka/sandbox/filters/firefox_monthly_dashboard.lua [hunk 1]
```diff
@@ -91,7 +91,7 @@ local function update_month(ts, cid, day_changed)
         current_month = month
     end
 
-    local country = fx.get_country_id(read_message("Fields[geo]"))
+    local country = fx.get_country_id(read_message("Fields[country]"))
```
Comment (mreid-moz) [inline heka/sandbox/filters/firefox_monthly_dashboard.lua:94]: Do we need to `s/geo/country/` on line 136 too? or is it called "geo" for compatibility?

File: heka/sandbox/filters/firefox_weekly_dashboard.lua [hunk 2]
```diff
@@ -81,7 +81,7 @@ local function update_week(ts, cid, day)
         error("data is in the past, this report doesn't back fill")
     end
 
-    local country = fx.get_country_id(read_message("Fields[geo]"))
+    local country = fx.get_country_id(read_message("Fields[country]"))
```
Comment (mreid-moz) [inline heka/sandbox/filters/firefox_weekly_dashboard.lua:84]: Do we need to `s/geo/country/` on line 121 too? or is it called "geo" for compatibility?

**Comment by trink** [comment]
geo is the column header used in the current dashboard (so the output should be 100% compatible/expected).

**Comment by mreid-moz** [comment]
Ok, let's keep it compatible.

**Pull request merged**